plugins/modules/zos_ping.rexx

```diff
@@ -1,7 +1,7 @@
 /* rexx  __ANSIBLE_ENCODE_EBCDIC__  */
 /* WANT_JSON */
 
-/* Copyright (c) IBM Corporation 2019, 2020, 2023 */
+/* Copyright (c) IBM Corporation 2019, 2023 */
```

Comment:
2024

Reply:
Done but no , it should be 2023 , the change was to move to the proper year structure for copy rights, where first year is when it was created, followed by a comma and then the last year it was edited; I edited in 2023 but never removed the middle year 2020.

Reply:
got it